src/components/card/card.js

```diff
@@ -8,7 +8,8 @@
 angular.module('material.components.card', [
     'material.core'
   ])
-  .directive('mdCard', mdCardDirective);
+  .directive('mdCard', mdCardDirective)
+  .directive('mdPlainImage', mdPlainImageDirective);
```

Comment:
@crisbeto, @topherfangio - we already have 

```html
<md-card-header>
  <md-card-avatar>
    <img class="md-user-avatar" src="avatar.png"/>
  </md-card-avatar>
</md-card-header> 
```
And we also have a `.md-card-image` class:

```html
<md-card>
 <img ng-src="card-image.png" class="md-card-image" alt="image caption">
</md-card>
```

So I think we should **not** use `.md-plain-image` and instead improve the **`md-card-image`** type style:

```css
md-card-image {
  display:block;
  position:relative;

  width: auto;
  max-width: 100%;
  height: auto !important; // !important because md-card also uses it.
}
```

```html
<md-card-image>
  <img ng-href="xxx" ></img>
</md-card-image>
```